apps/galaxy/src/modules/Orders/Details.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  DesignInformation,
+  QuotationDetails,
+  TileColor,
+  Order,
+} from "@src/services/orders/useOrders";
+import {
+  Button,
+  Image,
+  Text,
+  Link,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Table,
+  Tbody,
+  Tr,
+  Td,
+  TableContainer,
+  useDisclosure,
+} from "@cc/ui-chakra";
+import { CellContext } from "@tanstack/react-table";
+
+const Details = (info: CellContext<Order, DesignInformation>) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <>
+      <Button onClick={onOpen} variant="secondary">
+        details
+      </Button>
+      <Modal scrollBehavior={"inside"} isOpen={isOpen} onClose={onClose}>
+        <ModalContent>
+          <ModalHeader>Order Details</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <Text fontWeight="bold">Quotation</Text>
+            <Link href={info.getValue().quotation} color="blue">
+              click to view quotation here
+            </Link>
+            <Text>
+              quotation details:{" "}
+              {info
+                .getValue()
+                .quotationDetails.map(
+                  (quotationDetail: QuotationDetails) =>
```

Comment:
Since we already know the `typeof quatationDetails`, the `quotationDetail` passed in the map callback will have it's type inferred from `typeof quotationDetails`, read [this comment](https://github.com/courtcanva/cc-multiverse/pull/37#discussion_r1033465902) for more detail or visit [this link](https://www.typescriptlang.org/docs/handbook/type-inference.html) for ts doc.